app/guides/error-messages.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Displaying error messages
+description: Why to avoid adding validation errors in prototypes.
+order: 10
+tags:
+  - general
+---
+
+The NHS prototype kit does not currently include any code or feature that makes it easy to display [error messages](https://service-manual.nhs.uk/design-system/components/error-message) based upon the user’s answers (for example, leaving a field empty).
+
+Whilst you can add validation errors, to do so you will have to write your own JavaScript code.
+
+## Avoid adding validation errors
+
+In most scenarios, we recommend not adding validation errors to your prototype, as it can be a lot of effort for little value.
+
+Instead, in usability research you could observe whether users do anything which would cause an error.
+
+If you want to test whether your error messages make sense, you could add some alternative pages which always include them, and prompt the participant to try and understand the issue.
+
+When developers are building your service for real, you could create a separate document to list all the possible errors in your service along with the required logic and the error message to display.
```

Comment:
```suggestion
When developers are building your service for real, you could create a single page in the prototype as a demonstration of the general look and feel, and a separate document to list all the possible errors in your service along with the required logic and the error message to display.
```

Reply:
The design system will tend to only include simple examples and out of context of a real page. I like to do one page with more complex questions including a conditional field in error. The expected design of this isn't obvious from the design system (arguably it should be!).